aries-site/src/layouts/content/Example/CompareUseCases.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext, Text } from 'grommet';
+import { Close, Checkmark } from 'grommet-icons';
+
+export const CompareUseCases = ({ imageSrcGood, imageAltGood, captionGood, imageSrcBad, imageAltBad, captionBad }) => {
```

Comment:
Comments are mainly for Grommet best practices and conventions. I think we should use the existing Do/Dont component presentation as oppose to re-working this one, unless doing so for learning purposes.